app/scripts/controllers/persistentVolumeClaim.js

```diff
@@ -38,16 +38,20 @@ angular.module('openshiftConsole')
     var storageClass = $filter('storageClass');
     var watches = [];
 
+    var updateStorageClass = function(pvc) {
+      var storageClassName = storageClass($scope.pvc);
+      if (storageClassName !== _.get(storageClass, 'metadata.name')) {
+        DataService.get($scope.storageClassesVersion, storageClassName, {}).then(function(sc) {
+          $scope.isExpansionAllowed = (!sc || sc.allowVolumeExpansion) && pvc.status.phase === 'Bound';
```

Comment:
If there's no storage class name, isExpansionAllowed is never set to true. It never reaches this block. (I'm assuming a storage class is not required for resize.)

`sc` var itself should never be null here.